routes/live/previews/[...block].tsx

```diff
@@ -50,6 +50,26 @@ export default function Preview(props: PageProps<Page>) {
 
 const addLocal = (block: string): string =>
   block.startsWith("islands") && block.endsWith("tsx") ? `./${block}` : block;
+
+const getPropsFromRequest = async (req: Request) => {
+  const url = new URL(req.url);
+  if (req.method === "POST") {
+    const reader = req.body?.getReader();
+    reader?.releaseLock();
```

Comment:
this is necessary to be done twice? I think you can let only the last one

Reply:
No, can't because when call getReader, the reader is locked: https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream/getReader

Reply:
![image](https://github.com/deco-cx/live.ts/assets/15680320/cd3c867b-b2e5-414f-91d6-7fd08512a929)

Reply:
I dropped both